packages/core/addon/components/navi-visualizations/table.js

```diff
@@ -313,6 +326,13 @@ export default Component.extend({
   },
 
   actions: {
+    /**
+     * @action onUpdateReport
+     */
+    onUpdateReport() {
+      this.attrs.onUpdateReport(...arguments);
```

Comment:
Can we just pass `onUpdateReport` prop to the template?

Reply:
That's how I did it initially, see my comment about the bug in dashboards